modules/contact-form/js/gutenblock.jsx

```diff
@@ -0,0 +1,630 @@
+					</PanelBody>
+				</InspectorControls>
+				<div className="grunion-form">
+					{this.props.children}
+					<TextControl
+						className="button button-primary button-default grunion-submit-button"
+						value={ this.props.submit_button_text }
+						placeholder={ __( 'Submit »' ) }
+						onChange={ this.onChangeSubmit }
+					/>
+				</div>
+			</Fragment>
+		);
+	}
+}
+
+function GrunionFieldRequiredToggle( props ) {
+	return (
+		<ToggleControl
+			label={ __( 'Required' ) }
+			checked={ props.required }
+			onChange={ props.onChange }
+		/>
+	);
+}
+
+class GrunionFieldSettings extends Component {
+	constructor( ...args ) {
+		super( ...args );
+		this.onChangeRequired = this.onChangeRequired.bind( this );
+	}
+
+	onChangeRequired( x ) {
+		this.props.setAttributes( { required: x } );
+	}
+
+	render() {
+		return (
+			<InspectorControls>
+				<PanelBody title={ __( 'Field Settings', 'jetpack' ) }>
+					<GrunionFieldRequiredToggle
+						required={ this.props.required }
+						onChange={ this.onChangeRequired }
+					/>
+				</PanelBody>
+			</InspectorControls>
+		);
+	}
+}
+
+class GrunionFieldLabel extends Component {
+	constructor( ...args ) {
+		super( ...args );
+		this.onChangeLabel = this.onChangeLabel.bind( this );
+	}
+
+	onChangeLabel( x ) {
+		this.props.setAttributes( { label: x.target.value } );
+	}
+
+	render() {
+		return (
+			<Fragment>
+				<input
+					type='text'
+					value={ this.props.label }
+					className='grunion-field-label'
+					onChange={ this.onChangeLabel }
+				/>
+				{ this.props.required && <span className="required">{ __( '(required)' ) }</span> }
+			</Fragment>
+		);
+	}
+}
+
+function GrunionField( props ) {
+	return (
+		<Fragment>
+			<GrunionFieldSettings
+				required={ props.required }
+				setAttributes={ props.setAttributes }
+			/>
+			<div className="grunion-field">
+				<TextControl
+					type={ props.type }
+					label={ <GrunionFieldLabel
+						required={ props.required }
+						label={ props.label }
+						setAttributes={ props.setAttributes }
+					/> }
+					disabled={ true }
+				/>
+			</div>
+		</Fragment>
+	);
+}
+
+function GrunionFieldTextarea( props ) {
+	return (
+		<Fragment>
+			<GrunionFieldSettings
+				required={ props.required }
+				setAttributes={ props.setAttributes }
+			/>
+			<div className="grunion-field">
+				<TextareaControl
+					label={ <GrunionFieldLabel
+						required={ props.required }
+						label={ props.label }
+						setAttributes={ props.setAttributes }
+					/> }
+					disabled={ true }
+				/>
+			</div>
+		</Fragment>
+	);
+}
+
+function GrunionFieldCheckbox( props ) {
+	return (
+		<Fragment>
+			<GrunionFieldSettings
+				required={ props.required }
+				setAttributes={ props.setAttributes }
+			/>
+			<div className="grunion-field">
+				<CheckboxControl
+					label={ <GrunionFieldLabel
+						required={ props.required }
+						label={ props.label }
+						setAttributes={ props.setAttributes }
+					/> }
+					disabled={ true }
```

Comment:
I believe `disabled` would be the same as `disabled={ true }` here

Reply:
Done